test/libyul/objectCompiler/jump_tags.yul

```diff
@@ -31,6 +31,7 @@ object "Contract" {
 //   tag_7
 //   jumpi
 //     /* "source":82:87   */
+//   pop
 //   jump(tag_6)
```

Comment:
This is an example of the ``pop``s moving up from the function exit label. And an example where inlining ``jump(tag_6)`` would definitely be good.